hera_qm/tests/test_xrfi.py

```diff
@@ -26,22 +28,30 @@ def get_accuracy(f, rfi, verbose=VERBOSE):
         print '\t Found RFI: %1.3f\n\t False Positive: %1.3f' % (correctly_flagged, false_positive)
     return correctly_flagged, false_positive
 
+def fake_flags():
+    RFI = 10
+    SIZE = 100
+    fakeflags = np.random.randint(0,2,size=(SIZE,SIZE)).astype(bool)
+    return fakeflags
+    
 
 def plot_waterfall(data, f, mx=10, drng=10, mode='lin'):
-    if not PLOT:
-        return
+    #if not PLOT:
+    #    return
     plt.subplot(121)
-    capo.plot.waterfall(data, mode='lin', mx=10, drng=10)
+    plt.imshow(np.abs(data),aspect='auto',cmap='jet')
+#    capo.plot.waterfall(data, mode='lin', mx=10, drng=10)
     plt.colorbar()
     plt.subplot(122)
-    capo.plot.waterfall(f, mode='lin', mx=10, drng=10)
+    #capo.plot.waterfall(f, mode='lin', mx=10, drng=10)
+    plt.imshow(f,aspect='auto',cmap='jet')
     plt.colorbar()
     plt.show()
 
 
 def plot_result(f, rfi):
-    if not PLOT:
-        return
+    #if not PLOT:
```

Comment:
Why are these lines commented out, along with the calls to plotting in tests? This should all be handled by the PLOT global variable.